openedx/core/djangoapps/oauth_dispatch/dot_overrides.py

```diff
@@ -11,8 +11,10 @@
 from oauth2_provider.models import AccessToken
 from oauth2_provider.oauth2_validators import OAuth2Validator
 from pytz import utc
+import logging
```

Comment:
Group imports properly: https://www.python.org/dev/peps/pep-0008/#imports. You can do this quickly by running `isort` on this file.